### ISSUE TITLE
[Idea] Event Timer

### ISSUE THREAD

**Issue by doej1367** [InventivetalentDev/HypixelMagmaBossTimer#10]
I don't know where to tell you this but I've decided it would fit here the best.
Now that Skyblock will fill up with events from time to time I thought it would be a good idea to make an 'SkyBlock Event Timer' out of this (or start a new project that's up to you).
I thought about the addition of a timer for 
- the spooky festival (first was on the 27.10.19 06:35 AM EST and it is every 5d 3h)
- the new year festival
- the magma boss
- spider boss (I know it is impossible to implement but maybe one day with your mod...)
- black auction (always :55 so it is useless to implement but just so to make it complete)
- alpha wolf (one day when there is more data about it)
- dragons (just a data collection about which were when and what they dropped having placed how many eyes using the mod to collect it)
- any future events that will come to skyblock

**Comment by InventivetalentDev** [comment]
Already a step ahead! :P https://hypixel.inventivetalent.org/

**Comment by doej1367** [comment]
Yeeessss :)

**Comment by doej1367** [comment]
![grafik](https://user-images.githubusercontent.com/19228364/67639681-f2281d00-f8f2-11e9-8a73-63f92aa979af.png)
are you sure the Spooky festival timer is correct? 
Because the first festival was at 12:35 and the next should be + 5d3h 15:35 (3:35 pm not 4:35 pm).
I mean I could be mistake because Time Zones are confusing, but the Dark auction Timer is correct for me:
![grafik](https://user-images.githubusercontent.com/19228364/67639718-6ebafb80-f8f3-11e9-9c1d-43d1d603cd75.png)

**Comment by doej1367** [comment]
And another Idea befor I forget it: maybe add the _(October 27th 2019, 9:51:13 pm)_ not only for the next date but for the next 3 or so. This could be usefull for the Spooky Event to know when there is one you can attend

**Comment by InventivetalentDev** [comment]
> are you sure the Spooky Festival timer is correct?

Pretty sure it's correct; since it's every year just like the new year's event, it should be on the same interval, which is 5d4h from what I've observed.  




> not only for the next date but for the next 3 or so

Great idea! Gonna look into adding that :)

**Comment by doej1367** [comment]
https://hypixel.net/threads/hypixel-get-your-timing-right.2404240/page-2#post-17053909
But Jayavarmen said.. it would be... but if you say so I'm gonna beleve it:
![grafik](https://user-images.githubusercontent.com/19228364/67642665-4e4c6a80-f90e-11e9-9793-021c20172668.png)

**Comment by InventivetalentDev** [comment]
Yeah, not sure either. I tested the new year's timer a couple times and it always seems to match up properly with the in-game calendar time with the 5d4h interval 🤷‍♀

**Comment by doej1367** [comment]
https://hypixel.net/threads/skyblock-patch-0-7-5-winter-island-balance-changes.2500488/
And another event for you to add :)

**Comment by InventivetalentDev** [comment]
Working on it! :) 

https://hypixel.inventivetalent.org/skyblock-winter-timer/
